src/config/dynamic-theme-fixes.config

```diff
@@ -14610,7 +14640,7 @@ INVERT
 paramountnetwork.com
 
 CSS
-.module-container {
+body {
```

Comment:
Please explain what this fixes? The `.module-container` fix still seems to work.

Reply:
Hi @Gusted,

This fix is because on some pages, there are still some elements that aren't styled when darkreader is active.

With only .module-container:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/1766174/190859654-5bf04eb7-f4ff-404e-8b3d-3fdd5edac9f6.png">

Changing .module-container to body:
<img width="1676" alt="image" src="https://user-images.githubusercontent.com/1766174/190859628-815937eb-bcef-49bd-9587-a7999daee010.png">

Reply:
It's worth mentioning that paramountnetwork.com, as well as tvland.com, cmt.com, vh1.com, logotv.com and bet.com (in addition to cc.com and mtv.com from the global dark sites list) all use the same technology to power their website as southpark.cc.com does. Changing `.module-container` to `body` brings these sites fixes in-line with what southpark.cc.com was already using. I'm going to re-commit to list as many of these under one fix as possible considering this.